_posts/installation/2019-03-15-oss-to-armory.md

```diff
@@ -0,0 +1,121 @@
+---
+date: 2019-03-13
+title: Upgrading from OSS to Armory Spinnaker
+categories:
+   - Installation
+description: Upgrading from OSS to Armory Spinnaker
+type: Document
+---
+
+If you are currently on OSS Spinnaker and interested in upgrading to Armory Spinnaker, you can easily upgrade if you used Halyard to install your Spinnaker cluster (you can also easily downgrade back to OSS Spinnaker)
+
+At a high level, the process to do this is as follows:
+* Back up your existing Halyard configuration
```

Comment:
not super important for this PR, maybe another one
but if you add `kramdown` and set https://github.com/armory/documentation/blob/master/_config.yml#L26
you can use auto generated table of contents: https://github.com/armory/documentation/blame/master/_spinnaker/configure_kayenta.md#L7-L9